config/wallet-config.json

```diff
@@ -7,67 +7,78 @@
       "enabled": true,
       "hasFastCheckoutProcess": true,
       "name": "MoonPay",
-      "hasTradingFees": false
+      "hasTradingFees": true,
+      "hasUSAvailability": true
```

Comment:
Funky camelcase, and also can read as _has USA vailability_.

Avoid capitalising acronyms imo, recommend `hasUsaAvailability` (or spell the name fully)

Reply:
Ok, thanks, yeah tried several things but didn't like any of them. Haha. Will change.